Overview.src.html

```diff
@@ -1349,6 +1368,9 @@ <h4 id=terminology-headers>Headers</h4>
 <p class=note>The details of referrer policies are discussed in Referrer Policy.
 <span data-anolis-ref>REFERRER</span>
 
+<h3>Client Hints list</h3>
+
+<p>A <dfn title=concept-client-hints-list>client hints list</dfn> is a list of <a href=http://httpwg.org/http-extensions/client-hints.html#rfc.section.2.2.1>Client Hint tokens</a>, whose values are one of `<code title>dpr</code>`, `<code title>width</code>`, `<code title>viewport-width</code>`.
```

Comment:
Also, a client hint token doesn't have a value, right? It's a value. So "which is one of" seems better. And then `viewport-width` needs "or" before it.